saleor/order/actions.py

```diff
@@ -19,6 +17,8 @@
     update_order_status,
 )
 
+from ..domain_utils import transaction_domain_atomic
```

Comment:
Move this import  to be declared after `from ..core` and before `from ..extensions.manager`